src/utils/getFileName.js

```diff
@@ -0,0 +1,15 @@
+import { APPROACHES, getFormattedFilePrefixDate } from '../constants';
+
+export const getFileName = (item, triggeredFile) => {
+  if (item.approach === APPROACHES.QUERY) {
```

Comment:
it's not valid, approach is changed when bulk edit is started, you have query fields to understand how flow was started

Reply:
thanks, fixed